index.html

```diff
@@ -185,5 +185,25 @@ <h2 class="third-text"><a>let's connect</a></h2>
           </li>
         </ul>
     </section>
+    <section class="contact-form">
+        <ul>
+            <li><img id="ellipse-12-2" src="elements/ellipse-12-2.png" alt="icon-7"></li>
+            <li><img id="ellipse-5-w" src="elements/ellipse-5-white.png" alt="icon-8"></li>
+            <li><img id="ellipse-3-w" src="elements/ellipse-3-white.png" alt="icon-9"></li>
+            <li><img id="rect-4-d" src="elements/rectangle-4-dark.png" alt="icon-10"></li>
+            <li><img id="rect-7-2" src="elements/rectangle-7-2.png" alt="icon-11"></li>
+        </ul>
+        <h2 class="contact-title">Contact me</h2>
+       <h3 class="contact-subtitle">If you have an application you are interested in developing, 
+           a feature that you need built or a project that needs coding.
+            I’d love to help with it 
+        </h3>
+        <form id="form-enmanuel" action="https://formspree.io/f/form-enmanuel" method="post">
```

Comment:
- [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
          - action value should be similar to https://formspree.io/f/{form_id} (Currently you are not using your form_id, please register yourself on formspree and use your unique form id so that you'll receive the message sent from your portfolio)